ci/staging_packages_buildspec.yml

```diff
@@ -0,0 +1,20 @@
+# Build Spec for AWS CodeBuild CI
+
+version: 0.2
+env:
+  shell: bash
+
+phases:
+  install:
+    commands:
+      - apt-get update
+      - apt-get install sudo -y
+      - python ts_scripts/install_dependencies.py --cuda=cu102 --environment=dev
+
+  build:
+    commands:
+      - python3 binaries/conda/build_packages.py --install-conda-dependencies
+      - exec bash
+      - python3 binaries/build.py --staging
+      - cd binaries/
+      - python3 upload.py --upload-pypi-packages --upload-conda-packages 
```

Comment:
this script is for gpu. how about cpu/win?

Reply:
I've added one buildspec for CPU, but not for win. Although, binaries that are generated from GPU are built into *.whl, and conda packages that can work on CPU and win. One for windows can be added once it's tested on windows.